crates/router/src/core/errors/utils.rs

```diff
@@ -384,7 +384,7 @@ impl<T> ConnectorErrorExt<T> for error_stack::Result<T, errors::ConnectorError>
                 | errors::ConnectorError::DateFormattingFailed
                 | errors::ConnectorError::InvalidDataFormat { .. }
                 | errors::ConnectorError::MismatchedPaymentData
-                | errors::ConnectorError::InvalidWalletToken
+                | errors::ConnectorError::InvalidWalletToken { .. }
```

Comment:
Even here ConnectorError::InvalidWalletToken should be mapped to ApiErrorResponse::InvalidWalletToken